zerver/lib/exceptions.py

```diff
@@ -360,19 +360,21 @@ def __init__(self, message_id: int) -> None:
 
 class InvitationError(JsonableError):
     code = ErrorCode.INVITATION_FAILED
-    data_fields = ["errors", "sent_invitations", "license_limit_reached"]
+    data_fields = ["errors", "sent_invitations", "license_limit_reached", "max_limit_reached"]
 
     def __init__(
         self,
         msg: str,
         errors: List[Tuple[str, str, bool]],
         sent_invitations: bool,
         license_limit_reached: bool = False,
+        max_limit_reached: bool = False,
```

Comment:
Should this be `daily_limit_reached`, for clarity?

Reply:
Done